db/migrate/20220105085729_split_customers_name.rb

```diff
@@ -0,0 +1,22 @@
+class SplitCustomersName < ActiveRecord::Migration[6.1]
+  class Customer < ActiveRecord::Base
+  end
+
+  def change
+    add_column :customers, :first_name, :string
+    add_column :customers, :last_name, :string
+    rename_column :customers, :name, :backup_name
```

Comment:
I just thought of another problem here. When we rename the column it won't be available for the app code any more. The app code is only reloaded once the whole migration is finished. So we may have a few minutes in which checkouts would fail because the old app code can't find the `name` column.

We have two options here:

1. Deploy this with maintenance mode. Basically planned downtime.
2. Not rename the column.

I always like to avoid downtime. Is it essential that we rename this?

I'll add a commit to revert that and you can let me know your thoughts.

Reply:
I just had the same thought. We can leave it alone for now and remove it later.

Reply:
You missed the change in the `schema.rb` file though :stuck_out_tongue: 

I'll drop it now :+1: